data/src/common_characters.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+"""Creates json file containing list of "Common" for each language
+
+This module takes TSV files from data and return a json file for 
+each language TSV file in data/tsv that lists the unicode 
+typed "Common" characters that appear ineach language TSV file.
+"""
+import os
```

Comment:
Order should be:

One blank line after the docstring, then:

```
import os
import json
import unicodedata

from typing import Dict, Optional

import regex  # type: ignore
import unicodedataplus  # type: ignore

from codes import TSV_DIRECTORY_PATH
```

note:

* lexicographic sorting of `Dict` before `Optional`
* got rid of the type: ignore on `codes`...shouldn't be necessary?